packages/core/src/Services/socket-general.js

```diff
@@ -307,21 +307,25 @@ export default BinarySocketGeneral;
 const ResponseHandlers = (() => {
```

Comment:
what happens with multiple requests / multiple connections, especially when returning different values?

Reply:
the legacy socket is always single connection. so, no issues here.